eslint.config.mjs

```diff
@@ -0,0 +1,35 @@
+// eslint.config.js
```

Comment:
Is it possible to change the file type to `ts` because I have never seen us using `mjs` anywhere?

Reply:
[`ts` might be a bit more involved](https://eslint.org/docs/latest/use/configure/configuration-files#typescript-configuration-files). I switched to `.mjs` is so that we can use `import` syntax, cos `eslintrc.json` isn't supported on this eslint version. We're starting to support `.mjs` for bundles, it doesn't require any more dependencies or configurations because eslint supports it natively.